requirements/requirements.txt

```diff
@@ -1,26 +1,18 @@
-Werkzeug==1.0.1
-lightwood >= 23.8.1.0, < 23.8.2.0
```

Comment:
Now that lightwood is not a main req anymore, it looks like if a user installs "core" mindsdb only, there would be no way of creating models. Not saying this is bad, per se. After all we've been meaning to write a lightweight default ML engine for a while now (see #5089). However it would be good to highlight this in the PR description, seems important.

Reply:
Technically this statement is not true @paxcema in setup.py we have a handful of "default extras" to install which Lightwood is currently part of.  So when pip installing you will still get lightwood installed.

However, related to this, after discussion with Jorge I am writing up a discussion for us to discuss as a team removing ALL internal ML engines from our default pip package in a future PR to simplify our package, shrink our docker image(s), improve our potential adoption in open source, etc.  If/when we do THAT we will need to coordinate with our documentation to update it along side our code.

Reply:
Thanks for the explanation! I see now that "default extras" include Lightwood indeed.

We're good as long as there's awareness that moving Lightwood out of the default/core stack will have implications in terms of "ML ease of use" for the average user, so whenever we end up doing this it must be communicated properly, as you said.